### PR TITLE
Katib: Fix links for all examples

### PR DESCRIPTION
**Pull request by andreyvelich**
This PR: https://github.com/kubeflow/katib/pull/1691 introduces the new structure for Katib examples.
I made the appropriate doc changes.

/assign @gaocegege @johnugeorge @shannonbradshaw @kimwnasptd

### CODE REVIEW

**Comment by andreyvelich** [comment]
cc @tenzen-y @eliaskoromilas

File: content/en/docs/components/katib/hyperparameter.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ This example randomly generates the following hyperparameters:
 Check the experiment status:
 
 ```shell
-kubectl -n <YOUR_USER_PROFILE_NAMESPACE> get experiment random-example -o yaml
+kubectl -n <YOUR_USER_PROFILE_NAMESPACE> get experiment random -o yaml
```
Comment (tenzen-y) [inline content/en/docs/components/katib/hyperparameter.md:239]:
```suggestion
kubectl -n kubeflow-user-example-com get experiment random -o yaml
```

Should we specify namespace?

File: content/en/docs/components/katib/hyperparameter.md [hunk 2]
```diff
@@ -452,35 +447,35 @@ the Kubeflow's TensorFlow training job operator, TFJob:
...
    ```shell
-   kubectl apply -f tfjob-example.yaml
+   kubectl apply -f tfjob-mnist-with-summaries.yaml
    ```
 
 1. You can check the status of the experiment:
 
    ```shell
-   kubectl -n <YOUR_USER_PROFILE_NAMESPACE> get experiment tfjob-example -o yaml
+   kubectl -n <YOUR_USER_PROFILE_NAMESPACE> get experiment tfjob-mnist-with-summaries -o yaml
```
Comment (tenzen-y) [inline content/en/docs/components/katib/hyperparameter.md:456]:
```suggestion
   kubectl -n kubeflow-user-example-com get experiment tfjob-mnist-with-summaries -o yaml
```
Same comment.

File: content/en/docs/components/katib/hyperparameter.md [hunk 3]
```diff
@@ -494,16 +489,16 @@ using Kubeflow's PyTorch training job operator, PyTorchJob:
...
    ```shell
-   kubectl apply -f pytorchjob-example.yaml
+   kubectl apply -f pytorchjob-mnist.yaml
    ```
 
 1. You can check the status of the experiment:
 
    ```shell
-   kubectl -n <YOUR_USER_PROFILE_NAMESPACE> describe experiment pytorchjob-example
+   kubectl -n <YOUR_USER_PROFILE_NAMESPACE> describe experiment pytorchjob-mnist
```
Comment (tenzen-y) [inline content/en/docs/components/katib/hyperparameter.md:498]:
```suggestion
   kubectl -n kubeflow-user-example-com describe experiment pytorchjob-mnist
```
Same comment.

File: content/en/docs/components/katib/hyperparameter.md [hunk 4]
```diff
@@ -163,38 +163,38 @@ if you want to contribute to Katib UI.
...
-   curl https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1beta1/random-example.yaml --output random-example.yaml
+   curl https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1beta1/hp-tuning/random.yaml --output random.yaml
    ```
 
-1. Edit `random-example.yaml` and change the following line to use your Kubeflow
-   user profile namespace:
+1. Edit `random.yaml` and change the following line to use your Kubeflow
+   user profile namespace (e.g. `kubeflow-user-example-com`):
 
    ```shell
```
Comment (gaocegege) [inline content/en/docs/components/katib/hyperparameter.md:196]: Should we remove the shell here?

**Comment by andreyvelich** [comment]
@gaocegege @tenzen-y Thank you for the review.
I addressed your changes.

**Pull request merged**